### PR TITLE
Adds the Head of Pseudostaff

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

two people can play this game

## Why It's Good For The Game

how is hardcoding me into HOP not good for the game

## Changelog
:cl:
admin: Adds the Head of Pseudostaff role to MSay
/:cl:

### CODE REVIEW

File: fulp_modules/main_features/mentorhelp/mentorsay.dm [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
 	log_mentor("MSAY: [key_name(src)] : [msg]")
 	msg = keywords_lookup(msg)
-	if(mentor_datum?.is_contributor)
+	if(src.key == "SgtHunk")
```
Comment (JohnFulpWillard) [inline fulp_modules/main_features/mentorhelp/mentorsay.dm:15]:
```suggestion
	if(src.key == "johnwillard")
```

yeah sorry about this sudden retirement, a shame too

Comment (SgtHunk) [reply fulp_modules/main_features/mentorhelp/mentorsay.dm:15]: its so sad John Willard was found dead with 5 gunshots to his head
i didnt know suicide could be like that

**Comment by JohnFulpWillard** [comment]
hold on a bit im gonna see if i can do something to literally end you

**Review by JohnFulpWillard — APPROVED** [review]

**Comment by SgtHunk** [comment]
Updates EosStation.dmm...

**Pull request merged**